vizro-core/src/vizro/static/css/accordion.css

```diff
@@ -42,7 +42,8 @@
   overflow-anchor: none;
   background-color: inherit;
   letter-spacing: -0.014px;
-  padding: var(--spacing-04) var(--spacing-02);
+  padding-left: 0;
```

Comment:
Can you put padding-left `var(--spacing-02);` padding-left: 0; makes the page titles indented.

Reply:
We've decided to have no padding such that the page titles are aligned with the selector titles (see slack thread w/ J)